packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -615,6 +675,13 @@ export abstract class FluidDataStoreContext
 		this.channel!.setConnectionState(connected, clientId);
 	}
 
+	public setReadOnlyState(readonly: boolean): void {
+		this.verifyNotClosed("setReadOnlyState", false /* checkTombstone */);
+
+		this.channel?.setReadOnlyState?.(readonly);
+		this._deltaManager.setReadonly(readonly);
```

Comment:
Nice place to mention `// For back compat until DataStoreRuntime knows about our readonly getter`

Reply:
this is the place you mention in the below comment where `this.channel?.setReadOnlyState?.(readonly)` channel is the IFluidDatastore Channel/Runtime

Reply:
Oh, gosh.  Yeah there it is.